### PR TITLE
Slightly improve interface for Database class in db_mongo_read.py. De…

### PR DESCRIPTION
**Pull request by fschlueter**
…fault behaviour should be largely maintained. Only the default database name when connecting to a mongo client via the connection_string option changed

### CODE REVIEW

**Comment by fschlueter** [comment]
@sjoerd-bouma why is the documentation failing :(

File: NuRadioReco/detector/RNO_G/db_mongo_read.py [hunk 1]
```diff
@@ -35,7 +35,35 @@ def _impl(self, *method_args, **method_kwargs):
...
+            Specify mongo client. You have 5 options:
+
+                * `"env_pw_user"`: Connect to a server with the environmental variables
+                `mongo_server`, `mongo_user`, and `mongo_password`
+                * `"RNOG_public"`: Preconfigured connection to read-only RNO-G Hardware DB
+                * `"RNOG_test_public"`: Preconfigured connection to read-only RNO-G Test Hardware DB
+                * `"connection_string"`: Use environmental variable `db_mongo_connection_string` to
+                connect to mongo server
+                * `"mongodb*": Every string which starts with `"mongodb"` will be used to connect to
+                a mongo server
```
Comment (sjoerd-bouma) [inline NuRadioReco/detector/RNO_G/db_mongo_read.py:59]: This indentation isn't right. You shouldn't indent the list as a whole, and the bullet points that span multiple lines should be indented by two spaces. See [here](https://nu-radio.github.io/NuRadioMC/Introduction/pages/contributing.html#lists)

Comment (fschlueter) [reply NuRadioReco/detector/RNO_G/db_mongo_read.py:59]: Thanks, that was not clear for me by the error msg

**Comment by fschlueter** [comment]
HURRRRAY!

**Comment by fschlueter** [comment]
@sjoerd-bouma do you want to wait for @jakobhenrichs  or do you want to approve?

**Review by sjoerd-bouma — APPROVED** [review]

**Pull request merged**